### ISSUE TITLE
Lack of mobile support

### ISSUE THREAD

**Issue by marcio133** [yairtawil/ng-drag-drop-list#16]
**Mobile support**

The lib doest have mobile support, which makes it a little limited.

The directive only have listeners to mouse events, so adding listeners to touch events should solve the problem.

**Comment by dannygoncalves** [comment]
Agree, I have used the library which works fine for a project (angular CDK drag and drop doesn't play well with material tabs) but then hit this dead end on mobile.
